docs/sections/components/cli/index.md

```diff
@@ -1,10 +1,10 @@
-# Command Line Interface
+# Command Line Interface (CLI)
 
-`Distilabel` offers a [`CLI`][distilabel.cli.pipeline.utils] to initially *explore* and *rerun* `Pipelines`, let's take a look.
+`Distilabel` offers a [`CLI`][distilabel.cli.pipeline.utils] to _explore_ and _re-run_ existing [`Pipeline`][distilabel.pipeline.Pipeline] dumps, meaning that an existing dump can be explored to see the steps, how those are connected, the runtime parameters used, and also re-run it with the same or different runtime parameters, respectively.
 
 ## Available commands
 
-We have two commands under the `CLI` app, `distilabel pipeline`:
+The only available command as of the current version of `distilabel` is `distilabel pipeline`.
```

Comment:
Yes, because we'll add more, but the first line is there to state that atm we only have one, and renaming the section to `Available command` doesn't make much sense 👍🏻